core/src/main/resources/filodb-defaults.conf

```
@@ -318,6 +318,9 @@ filodb {
 
   query {
 
+    # feature flag to enable start time inclusive in window lookback
+    inclusive_range = true
```

I think all of our filodb configs are `kebab-case`:
```suggestion
    inclusive-range = true
```